frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-api.ts

```diff
@@ -7,4 +7,53 @@ export const exposePluginAPI = () => {
     useK8sWatchResources: require('@console/internal/components/utils/k8s-watch-hook')
       .useK8sWatchResources,
   };
+  window.internalAPI = {
```

Comment:
Oh so ugly. @rawagner @vojtechszocs are we planning to move away from global variables in 4.9?

Reply:
@christianvogt Yes, this should be done as part of #9432